fontforgeexe/wordlistparser.c

```diff
@@ -284,15 +284,21 @@ u_WordlistEscapedInputStringToRealString_readGlyphName(
 	    {
 		unichar_t* endptr = 0;
 		long unicodepoint = u_strtoul( glyphname+3, &endptr, 16 );
+                SplineChar* tmp = 0;
```

Comment:
This pointer ought to be null. The existing pointer before probably ought to be null as well.